### ISSUE TITLE
THUCNews，acc很低（0.027177358580844267）大概怎么回事 。我的参数如下

### ISSUE THREAD

**Issue by easywaytodo** [ymcui/Chinese-BERT-wwm#69]
#!/usr/bin/env bash

epochs="3 10 15 50 100"
batchs="16 32 64"
#
#epochs="1"
#batchs="8"
task_n="thucnews"
task_names="chnsenticorp weibo thucnews"
data_dirs="data weibo_senti_100k THUCNews"
#list="rootfs usr data data2"
python thucnews_preprocess.py
for epoch in $epochs;
do

echo epoch= $e  ;
for batchsize in $batchs;
do
echo  "         "batch= $batchsize  ;
rm -rf tmp

time=$(date "+%Y%m%d-%Hh%Mm%Ss")

echo "start    "$task_n" :"$time"----epoch"$epoch"bacth"$batchsize >> $task_n-$time"epoch"$epoch"bacth"$batchsize".txt"
echo "\n\n" >> $task_n-$time"epoch"$epoch"bacth"$batchsize".txt"
python3 run_classifier.py \
  --data_dir=data/THUCNews \
  --task_name=$task_n \
  --vocab_file=publish/vocab.txt \
  --bert_config_file=publish/bert_config.json \
  --output_dir=tmp/sim_model \
  --do_train=true \
  --do_eval=true \
  --init_checkpoint=publish/bert_model.ckpt \
  --max_seq_length=128 \
  --train_batch_size=$batchsize \
  --learning_rate=5e-5 \
  --num_train_epochs=$epoch >> $task_n-$time"epoch"$epoch"bacth"$batchsize".txt"

python3 run_classifier.py \
  --task_name=$task_n \
  --do_predict=true \
  --data_dir=data/THUCNews \
  --vocab_file=publish/vocab.txt \
  --bert_config_file=publish/bert_config.json \
  --init_checkpoint=tmp/sim_model \
  --max_seq_length=128 \
  --output_dir=tmp/sim_model  >> $task_n-$time"epoch"$epoch"bacth"$batchsize".txt"

python3 thuc_test.py  >> $task_n-$time"epoch"$epoch"bacth"$batchsize".txt"

done
done

**Comment by easywaytodo** [comment]
测试结果如下
(base) y3$cat thucnews-20191101-12h11m34sepoch10bacth32.txt
start    thucnews :20191101-12h11m34s----epoch10bacth32
\n\n
f.readlines()= 167198
label_size= 167198
label_indexs_size= 167198
test: 8
label: 1
test1: 8
label1: 1
test2: 8
label2: 1
np.array(label).shape= (167198,)
np.array(flag).shape= (167198,)
              precision    recall  f1-score   support

           0       0.00      0.00      0.00     10168
           1       0.00      0.00      0.00      4010
           2       0.00      0.00      0.00      4874
           3       0.00      0.00      0.00      2672
           4       0.00      0.00      0.00     32584
           5       0.00      0.00      0.00     26320
           6       0.00      0.00      0.00      6516
           7       0.00      0.00      0.00       714
           8       0.05      1.00      0.10      8386
           9       0.00      0.00      0.00     12616
          10       0.00      0.00      0.00      7418
          11       0.00      0.00      0.00      1516
          12       0.00      0.00      0.00     18526
          13       0.00      0.00      0.00     30878

    accuracy                           0.05    167198
   macro avg       0.00      0.07      0.01    167198
weighted avg       0.00      0.05      0.00    167198

acc is:0.05015610234572184
(base) y3$ cat thucnews-20191102-07h03m37sepoch10bacth64.txt
start    thucnews :20191102-07h03m37s----epoch10bacth64
\n\n
f.readlines()= 167198
label_size= 167198
label_indexs_size= 167198
test: 8
label: 1
test1: 11
label1: 1
test2: 11
label2: 1
np.array(label).shape= (167198,)
np.array(flag).shape= (167198,)
              precision    recall  f1-score   support

           0       0.00      0.00      0.00     10168
           1       0.00      0.00      0.00      4010
           2       0.00      0.00      0.00      4874
           3       0.00      0.00      0.00      2672
           4       0.27      0.00      0.00     32584
           5       0.00      0.00      0.00     26320
           6       0.00      0.00      0.00      6516
           7       0.00      0.00      0.00       714
           8       0.06      0.47      0.11      8386
           9       0.00      0.00      0.00     12616
          10       0.00      0.00      0.00      7418
          11       0.01      0.41      0.01      1516
          12       0.00      0.00      0.00     18526
          13       0.00      0.00      0.00     30878

    accuracy                           0.03    167198
   macro avg       0.02      0.06      0.01    167198
weighted avg       0.06      0.03      0.01    167198

acc is:0.027177358580844267

**Comment by ymcui** [comment]
#57
